frontend/public/components/modals/expand-pvc-modal.jsx

```diff
@@ -1,18 +1,21 @@
 import * as React from 'react';
 
 import { createModalLauncher, ModalTitle, ModalBody, ModalSubmitFooter } from '../factory/modal';
-import { PromiseComponent, RequestSizeInput, resourceObjPath, history } from '../utils';
+import { PromiseComponent, RequestSizeInput, resourceObjPath, history, validate } from '../utils';
 import { k8sPatch, referenceFor } from '../../module/k8s/';
 
+import { getPvcStorageSize } from '@console/shared';
+
 // Modal for expanding persistent volume claims
 class ExpandPVCModal extends PromiseComponent {
   constructor(props) {
     super(props);
+    const defaultSize = validate.split(getPvcStorageSize(props.resource));
```

Comment:
Do we need to handle missing values? It looks like these fields are optional in the API.

Reply:
Do we want to prefill the requested capacity or the actual capacity from `status`?

Reply:
It's the requested size. Because what you pass in the pvc object is requested capacity. So even if the requested capacity is not fulfilled out it will not throw error next time we try to expand.

That's the same we did with OCP 3.11 also but this was the debt that got left.